corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -67,6 +67,20 @@ FormplayerFrontend.reqres.setHandler('clearMenu', function () {
     $('#menu-region').html("");
 });
 
+$(document).bind("ajaxStart", function(){
+    var requestElements = $(".formplayer-request");
+    for (var i = 0; i < requestElements.length; i++ ) {
+        requestElements[i].style.pointerEvents = 'none';
+    }
+    tfLoading();
```

Comment:
When I remove this call the blue bar doesn't display - where's this call made? I can't see it in webformsession